packages/ipfs/src/http/api/routes/webui.js

```diff
@@ -1,36 +1,31 @@
 'use strict'
 
-const multiaddr = require('multiaddr')
 const debug = require('debug')
+const { gateway } = require('../../gateway/resources')
 const log = debug('ipfs:webui:info')
 log.error = debug('ipfs:webui:error')
 
+const webuiCid = 'bafybeigxqbvc6qxk2wkdyzpkh7mr7zh5pxbvpjb6a6mxdtpwhlqaf4qj5a' // v2.7.4
```

Comment:
@jessicaschilling  in the future, when we release new ipfs-webui we will need to update only this one line :crossed_fingers:

Reply:
@lidel Is this the new version of the (broken link) js-ipfs file that appears here? https://github.com/ipfs-shipyard/ipfs-webui#releasing-a-new-version-of-the-webui

Reply:
@jessicaschilling  Yes. I'll update that README when I finish the release dance for v2.7.4